src/effects/VST/VSTEffect.cpp

```diff
@@ -1299,20 +1299,30 @@ std::unique_ptr<EffectUIValidator> VSTEffect::PopulateUI(ShuttleGui &S,
       mGui = true;
    }
 
+   auto pParent = S.GetParent();
+   pParent->PushEventHandler(this);
+
+   auto validator = std::make_unique<VSTEffectValidator>(dynamic_cast<VSTEffectInstance&>(instance), *this, access, pParent);
+
+   // These could be moved in the validator constructor
+   validator->Load();
+   validator->mParent = mParent;
+   validator->mTimer = std::make_unique<VSTEffectTimer>(this);
+   validator->mDialog = static_cast<wxDialog*>(wxGetTopLevelParent(parent));
```

Comment:
These three assignments can be done instead in the constructor of the validator

Reply:
Done (actually, by using already passed arg `parent`) - and I also moved there the call to `Load()`.